Tools/autotest/common.py

```diff
@@ -5300,9 +5318,20 @@ def test_rate(self, desc, in_rate, expected_rate):
         else:
             set_interval = self.rate_to_interval_us(in_rate)
 
-        self.mavproxy.send("long SET_MESSAGE_INTERVAL %u %d\n" %
-                           (self.victim_message_id, set_interval))
-        self.mav.recv_match(type='COMMAND_ACK', blocking=True)
+        if False:
```

Comment:
why if False ? just add an arguement : with_mavproxy=False

Reply:
This will be replace by a `self.using_mavproxy_connection()` soon, from the sitl-direct branch.